TheAurorian/src/main/resources/assets/theaurorian/patchouli_books/the_aurorian_guide/zh_cn/entries/magicalitems/umbrapickaxe.json

```
@@ -0,0 +1,13 @@
+{
+    "name": "本影镐",
+    "icon": "theaurorian:umbrapickaxe",
+    "category": "3magicalitems",
+    "readbydefault": true,
+    "pages": [
+        {
+            "type": "spotlight",
+            "item": "theaurorian:umbrapickaxe",
+            "text": "$(#008cff)本影镐$()能让你选择一种方块并获得挖掘它时的速度增益，但选择这一过程会消耗大量耐久度。潜行单击一个方块会移除当前选中的方块。"
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
            "text": "$(#008cff)本影镐$()能让你选择一种方块并获得挖掘它时的速度增益，但选择这一过程会消耗大量耐久度。"
```
删掉的这句话需要实测。

---

无问题。

```suggestion
            "text": "$(#008cff)本影镐$()能让你选择一种方块，并获得挖掘这种方块时获得速度增益，但选择这一过程会消耗大量耐久度。潜行单击一个方块会移除当前选中的方块。"


```

---

冗余。